app/controllers/lit/api/v1/base_controller.rb

```diff
@@ -3,7 +3,7 @@ module Api
     module V1
       class BaseController < ActionController::Base
         layout nil
-        respond_to :json
+        respond_to :json if ::Rails::VERSION::MAJOR < 5
```

Comment:
why?

Reply:
https://stackoverflow.com/questions/35983628/undefined-instance-method-respond-to-in-rails-5-api-controller